docs/troubleshoot-ceph-crashcollector.md

```diff
@@ -0,0 +1,53 @@
+# Troubleshoot rook-ceph crashcollector
+
+If you enable the crash collector in your rook-ceph installation, you might see some error logs in the crashcollector pods :
+
+```bash
+kubectl logs -f rook-ceph-crashcollector-dev-sc-storage-0-6695c6zx8rg
+INFO:ceph-crash:monitoring path /var/lib/ceph/crash, delay 600s
+WARNING:ceph-crash:post /var/lib/ceph/crash/2023-11-21T22:05:00.157537Z_cd3d06ca-bc60-4b24-9c05-998876d4f209 as client.crash.rook-ceph-crashcollector-dev-sc-storage-0-6695c6zx8rg failed: 2023-11-21T22:20:07.837+0000 7f8489b12700 -1 monclient(hunting): handle_auth_bad_method server allowed_methods [2] but i only support [2,1]
+2023-11-21T22:20:07.837+0000 7f848a313700 -1 monclient(hunting): handle_auth_bad_method server allowed_methods [2] but i only support [2,1]
+2023-11-21T22:20:07.837+0000 7f8489311700 -1 monclient(hunting): handle_auth_bad_method server allowed_methods [2] but i only support [2,1]
+[errno 13] RADOS permission denied (error connecting to the cluster)
+```
+
+Note that in most cases, those error messages are expected due to the rook-ceph crash posting implementation.
+
+Basically, what happens is that Ceph [scans](https://github.com/ceph/ceph/blob/main/src/ceph-crash.in#L66) the crash folder */var/lib/ceph/crash* every *X* minutes, by default the delay between scans is 10 minutes. Whenever it find crashes, it will try to [post](https://github.com/ceph/ceph/blob/main/src/ceph-crash.in#L44) them, once the post action succeeds, ceph will [move those crashes](https://github.com/ceph/ceph/blob/main/src/ceph-crash.in#L84) to the **posted** folder.
```

Comment:
```suggestion
Basically, what happens is that Ceph [scans](https://github.com/ceph/ceph/blob/main/src/ceph-crash.in#L66) the crash folder `/var/lib/ceph/crash` every *X* minutes, by default the delay between scans is 10 minutes. Whenever it find crashes, it will try to [post](https://github.com/ceph/ceph/blob/main/src/ceph-crash.in#L44) them, once the post action succeeds, Ceph will [move those crashes](https://github.com/ceph/ceph/blob/main/src/ceph-crash.in#L84) to the **posted** folder.
```

I would say prefer using inline code snippets for paths as it makes them a bit clearer.